10-VM-Ubuntu-2-NodeType-Secure/AzureDeploy.json

```diff
@@ -813,7 +813,8 @@
                                             "x509StoreName": "[parameters('certificateStoreValue')]"
                                         }
                                     },
-                                    "typeHandlerVersion": "1.1"
+                                    "enableAutomaticUpgrade": true,
```

Comment:
This should be in the "properties" section above. Same level as "autoUpgradeMinorVersion"